apps/create-vesting/src/app/page.tsx

```diff
@@ -1,126 +1,162 @@
 "use client";
 
-import {
-  type CoWHookDappActions,
-  type HookDappContext,
-  initCoWHookDapp,
-} from "@cowprotocol/hook-dapp-lib";
-
 import {
   Input,
   PeriodWithScaleInput,
   ButtonPrimary,
   ContentWrapper,
   TokenAmountInput,
   Wrapper,
+  HookDappContextAdjusted,
 } from "@bleu/cow-hooks-ui";
+import { Token } from "@uniswap/sdk-core";
 
-import { useEffect, useState } from "react";
-
+import { useCallback, useMemo } from "react";
+import { useIFrameContext, Spinner } from "@bleu/cow-hooks-ui";
```

Comment:
Import all the components from the same package together. You already imported `@bleu/cow-hooks-ui`. We should add the eslint rules once we have sometime 😅